### PR TITLE
Remove underscore

### PR DESCRIPTION
**Pull request by harryadel**
I tried to apply what we learned in https://github.com/meteor/blaze/pull/331
Relevant issues: https://github.com/meteor/meteor-feature-requests/issues/48 
https://github.com/meteor/meteor/issues/1009#issuecomment-307090385

### CODE REVIEW

File: docs/_config.yml
Comment (StorytellerCZ) [inline docs/_config.yml:77]: I think for docs you will also need to replace underscore in the Meteor theme.

File: packages/browser-policy-content/browser-policy-content.js
Comment (StorytellerCZ) [inline packages/browser-policy-content/browser-policy-content.js:68]: Maybe we could use includes here. Should be a tiny bit faster as includes returns boolean.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]
Check the first few files. Maybe it might be better to split into multiple smaller PRs to make it more manageable.
Other than that a great job!

**Comment by harryadel** [comment]
> Maybe it might be better to split into multiple smaller PRs to make it more manageable.

It's just there're way too many packages involved with underscore and it'd be discouraging/monotonous to tackle them one by one.

File: docs/_config.yml
Comment (harryadel) [reply docs/_config.yml:77]: Sorry but I don't understand. Replace underscore as in some config file or Meteor theme code?

**Comment by afrokick** [comment]
> > Maybe it might be better to split into multiple smaller PRs to make it more manageable.
> 
> It's just there're way too many packages involved with underscore and it'd be discouraging/monotonous to tackle them one by one.

Agree with you.

But as more files changed in PR, as less chance to be merged)

**Comment by harryadel** [comment]
>But as more files changed in PR, as less chance to be merged)

I understand the value of small PRs and I'm not against them but given the scope of underscore and it's related changes I thought it'd be better to be tackled at once just like https://github.com/meteor/blaze/pull/331

This PR has been time intensive as is so I'm willing to solve any comments on this PR but if it's a no go maybe someone else can pick it up and use this PR as a reference.

**Comment by StorytellerCZ** [comment]
If we are going to merge this as one PR, then 2.6 release is a good candidate and we need to do it soon so that all changes follow from that otherwise we will spiral into endless merge resolutions.

**Comment by harryadel** [comment]
@StorytellerCZ like I said, I'm down to solve any comments you guys have but https://github.com/meteor/blaze/pull/359 needs to be merged to fix the tests.

**Comment by filipenevola** [comment]
Hi @harryadel, the tests are not passing, and also you are changing multiple packages.

I know the change is the same, but multiple PRs are necessary to get good coverage in the reviews.

As our [Contribution](https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#proposing-your-change) guidelines say:

> Split features up into smaller, logically separate chunks. It is unlikely that large and complicated PRs will be merged.

You can keep a single branch for your work, but please open different PRs for each package, ok?

**Comment by harryadel** [comment]
We have to wait for the release of 2.6 to retest things as it's failing because of the unavailability of the blaze packages. 
```js
   buf: '=> Running Meteor from a checkout -- overrides project version (Meteor 1.10.1)\n' +
        '=> Errors while initializing project:\n' +
        '\n' +
        'While selecting package versions:\n' +
        'error: No version of blaze-html-templates satisfies all constraints: @1.0.4, @=2.0.0\n' +
        'Constraints on package "blaze-html-templates":\n' +
        '* blaze-html-templates@1.0.4 <- top level\n' +
        '* blaze-html-templates@=2.0.0 <- top level\n' +
        '\n',
      fullBuffer: '=> Running Meteor from a checkout -- overrides project version (Meteor 1.10.1)\n' +
        '=> Errors while initializing project:\n' +
        '\n' +
        'While selecting package versions:\n' +
        'error: No version of blaze-html-templates satisfies all constraints: @1.0.4, @=2.0.0\n' +
        'Constraints on package "blaze-html-templates":\n' +
        '* blaze-html-templates@1.0.4 <- top level\n' +
        '* blaze-html-templates@=2.0.0 <- top level\n' +
        '\n',
```

**Comment by diavrank** [comment]
Hi, have already considered updating this package `meteorhacks:ssr`? this package is very used to build email templates and I think it is using the `underscore` package.

**Comment by StorytellerCZ** [comment]
@diavrank that is not a core package, so it is out of scope here. You should contact the package maintainers for that.

**Comment by diavrank** [comment]
@StorytellerCZ the thing here is that it is a unmaintained package but it is still used by many developers. Can we have a compat package for that?

**Comment by Grubba27** [comment]
> the thing here is that it is a unmaintained package but it is still used by many developers. Can we have a compat package for that?

Hey, @diavrank could you make this request an issue? so that we can discuss it and be trackable :)

**Pull request closed without merging**
